components/lib/fileupload/FileUpload.d.ts

```diff
@@ -130,6 +130,7 @@ export declare class FileUpload extends React.Component<FileUploadProps, any> {
     public upload(): void;
     public clear(): void;
     public formatSize(bytes: number): number;
+    public onFileSelect(event: FileUploadSelectParams): boid;
```

Comment:
Should this be `void`?

Reply:
Yep this should have been caught.  THe TypeScript checker was not doing its job because of a setting I just found and fixed and it then found a bunch of other issues